### ISSUE TITLE
Wrong deprecation message with mergeFeatures

### ISSUE THREAD

**Issue by antagomir** [microbiome/mia#416]
We get wrong warning:

```
library(mia)
data(GlobalPatterns, package="mia")
tse <- GlobalPatterns 
mergeFeatures(tse, f="Genus")
```

> 'mergeRows' is deprecated.
> Use 'mergeFeatures' instead.

-> We should get this deprecation message only when mergeRows is called, not when mergeFeatures is called.

-> To check also for other deprecated functions.

**Comment by TuomasBorman** [comment]
Is this done? @ake123

**Comment by ake123** [comment]
it was done!

**Comment by antagomir** [comment]
Can you close issues that you have solved (like this one?).

**Comment by ake123** [comment]
It looks like this fix is being reset for some reason so I need to fix it again

**Comment by ake123** [comment]
Now it is fixed
